packages/samples/headless-ssr-commerce/e2e/search/search.spec.ts

```diff
@@ -0,0 +1,196 @@
+import {JSDOM} from 'jsdom';
```

Comment:
I'm usually really against JSDOM (because its API is quite incomplete compared to a full-fledged browser).

However, here, it makes good sense 👍 Nice idea.